src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -218,7 +218,7 @@ private function displayGeneralText(SymfonyStyle $io, ?string $filter = null): v
         $types = ['functions', 'filters', 'tests', 'globals'];
         foreach ($types as $index => $type) {
             $items = [];
-            foreach ($this->twig->{'get'.ucfirst($type)}() as $name => $entity) {
+            foreach ((new \ReflectionMethod(Environment::class, 'get' . ucfirst($type)))->invoke($this->twig) as $name => $entity) {
```

Comment:
Would it be possible to rely on the Twig public API instead ? Private methods of Twig are not covered by the BC promise.

Reply:
we possibly could, but that would require to completely rewrite the command, for functions, filters and tests we also depend on internal methods of the `Environment` class

Reply:
Maybe I can revert the change from `public` to `private` for `getGlobals`? We will still use internal methods, but at least that works without using reflection.

Reply:
that would be even better I guess

Reply:
I have reverted the change here

Reply:
FTR, `getGlobals()` is now also no longer internal since twigphp/Twig@194cced47b766fb49ac587ff7d4a02529d1431d4